source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -16,6 +16,16 @@ Overview
 In this guide, you can learn how to use bulk operations in the
 MongoDB Java Driver. 
 
+.. note:: Client Bulk Write Method
+
+   This guide primarily focuses on the ``MongoCollection.bulkWrite()``
+   method, which allows you to perform bulk operations on a collection. 
+   When connecting to a deployment running {+mdb-server+} 8.0 or later,
+   you can use the ``MongoClient.bulkWrite()`` method to write
+   to multiple databases and collections in the same call. See the 
+   :ref:`java-sync-client-bulk-write` section below to learn how to use the
+   ``MongoClient.bulkWrite()`` method.
```

Comment:
S: on second thought, since this guide covers both types of bulk writes, maybe you can just remove this note and change the content of the overview instead. For example, you can revise the overview section to say something like 
"You can perform bulk write operations by using the Java driver to perform multiple data changes in one call. You can perform bulk operations at the following levels:

- collection level: xxxx
- client level: xxxx

And then add some information about the apis and link to the appropriate sections